### ISSUE TITLE
Processing in parallel

### ISSUE THREAD

**Issue by cheerfulstoic** [cwbriones/scientist#2]
Hey!

I just started using the `scientist` gem and I was curious if there was something for Elixir.  I'm glad you've already done a lot of work in this.  One thing that I was curious about: Have you thought about the possibility of making experiments run in parallel?  With Elixir you could make the control and candidate blocks run in separate processes so as not to duplicate the time taken.  There would potentially be consequences for copying large amounts of data between processes, but it could be a nice option
